draft-ietf-rats-architecture.md

```diff
@@ -1424,30 +1425,57 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
+Handles are opaque, non-predictable data regularly broadcast by a Handle Distributor.  
+Each handle represents a specific epoch,
+in which generated Evidence can be asserted to be fresh.
 
-In this example, Evidence generation based on received handles always uses the current (most recent) handle.
-As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
-Due to distribution over the network, there is some jitter with respect to the time the Handle is received, time(HR), for each involved entity.
-To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.
+The distribution of a new handle determines the transition from one epoch to
+the next.  The transition between epochs goes through an "overlap" phase in
+which the previous epoch is still extant while the current epoch is
+established.  The span of the overlap phase is determined solely by the
+transmission delay of the underlying communication channel,
+`time(HR)-time(HT)`.  Once the overlap phase is over, an epoch is fully
+established.
 
-From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HR'_v)-time(HR_v)) - (time(HR_a)-time(EG_a)) \< validity-duration".
+In order to determine freshness of Evidence or Attestation Results, consumers
+of messages based on handles need to understand whether an epoch is fully
+established or in overlap phase.
 
-Due to the validity-duration of handles, multiple different pieces of Evidence can be generated based on the same handle.
-The resulting granularity (time resolution) of Evidence freshness is typically lower than the resolution of clock-based tickcounters.
+The Handle Distributor, hosted in a dedicated entity or co-located with other
+RATS roles, is trusted by the protocol participants to behave correctly with
+regards to its duty of handle distribution.
 
-The following example illustrates a hypothetical Background-Check Model solution that uses handles and requires a trustworthy time source available to the Handle Distributor role.
+Consumers of Conceptual Messages based on handles must be able to safely trace
+each handle's origin back to the broadcasting Handle Distributor either using
+topological considerations (e.g., co-location), or by cryptographic means.
+
+The example in {{fig-handle-passport}} illustrates a Handle-based interaction
+using the passport topology model.
```

Comment:
This is either redundant with the first sentence of this section, or everything above here should be moved to the freshness section.